### ISSUE TITLE
422 Unprocessable Entity when trying to login with complete new setup

### ISSUE THREAD

**Issue by marvinpuethe** [sleede/fab-manager#300]
**Describe the bug**
We are trying to setup the fabmanager environment using Ubuntu Server 20.04 on Hyper-V Windows 10 or Hyper-V Server. When trying to do any action like sign_in, confirmation, register the server responds with 422 Unprocessable Entity:

> /users/sign_in.json:1 Failed to load resource: the server responded with a status of 422 (Unprocessable Entity)

We did the preparations described under production_readme.md manual or with the script prepare-vps.sleede.com and ran the setup with the script from setup.fab.mn.
We are using the default nginx container without lets encrypt. The env variables for smtp are configured with Office 365 and the localization variables are set to german.

**To Reproduce**
1. Install Ubuntu Server 20.04 on Hyper-V Windows 10 or Hyper-V Server
2. Do the installation described under [production_readme.md](https://github.com/sleede/fab-manager/blob/master/doc/production_readme.md)
3. When asked how to deploy select yes for nginx, no for lets encrypt. Configure [SMTP for Office 365](https://support.microsoft.com/de-de/office/pop-imap-und-smtp-einstellungen-8361e398-8af4-4e97-b147-6c6c4ac95353). Set the localization variables to german (de/de-DE/de-de) described at the various links.
4. Open Fabmanager and try to login with the user defined in the setup process

**Expected behavior**
Login and Register should work and not return 422 Unprocessable Entity

**Screenshots**
![image](https://user-images.githubusercontent.com/22427295/132940398-0d18f4df-8985-4a50-b89c-7e39ca17b732.png)

![image](https://user-images.githubusercontent.com/22427295/132940408-2924359d-f384-4773-b46c-920d8142935e.png)

**Server (please complete the following information):**
 - OS: Hyper-V / Ubuntu 20.04
 - Kernel: 
![image](https://user-images.githubusercontent.com/22427295/132940445-87dd3245-e20f-46a3-90cf-d846a2171c00.png)
 - Fab-manager version: 5.1.6

**Browser (please complete the following information):**
 - Name: Google Chrome/Microsoft Edge
 - Version:  93.0.4577.63/93.0.961.44

**Additional context**
-

**Comment by gnepud** [comment]
Hi @marvinpuethe 

Do you have any errors on sign up a user ?
User marvin.puethe@hrw-fablab.de has created in database ?

You can try access your container of fabmanager then find this user 
```
docker exec -it <fabmanager_contrainer> bash
rails console
User.findBy(email: 'marvin.puethe@hrw-fablab.de')
```

**Comment by sylvainbx** [comment]
Hi @marvinpuethe 
You said that you are not using let's encrypt. Does it mean that you have your own SSL certificate or that you are only in HTTP (without encryption)?

**Comment by marvinpuethe** [comment]
Hi together and thanks for your response,

@sylvainbx
Currently we are in a dev environment and only serving over http without lets encrypt because it is not facing the internet.

@gnepud 
I tried to access the container and run "rails console" but it said `bash: rails: command not found`
I also tried installing ruby from debian packages via `apt update && apt install ruby ruby-dev build-essentials` and installed rails via `gem install rails`. But `rails console` only opens the usage help.

Greetings
Marvin
